src/routes/CreateEvent.js

```diff
@@ -13,42 +15,170 @@ const CreateContainer = styled('div')`
   flex-direction: column;
 `
 
+const LockedContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+`
+
 const TextInput = styled(DefaultTextInput)`
   margin-bottom: 20px;
 `
 
-class Create extends Component {
-  state = {
-    password: ''
-  }
+const Button = styled(DefaultButton)`
+  margin: 60px auto 0px auto;
+`
 
-  render() {
-    const { password } = this.state
-
-    return (
-      <CreateContainer>
-        <PartyForm
-          onCompleted={this._onCreated}
-          mutation={CREATE_PENDING_PARTY}
-          variables={{ password }}
-          type="create"
-        >
-          <Label>SECRET PASSWORD:</Label>
-          <TextInput
-            value={password}
-            onChangeText={val => this.setState({ password: val })}
-            type="password"
-          />
-        </PartyForm>
-      </CreateContainer>
-    )
-  }
+const UnlockCredit = styled('div')`
+  margin: 40px auto 140px auto;
+  font-size: 14px;
+`
+
+const UnlockedLogo = styled('a')`
+  color: #ff6771;
+  font-weight: 800;
+`
+
+const AddrDisplay = styled('span')`
+  background-color: #6e76ff;
+  color: #fff;
+  border-radius: 2px;
+  padding: 5px;
+  font-size: 10px;
+  margin-right: 10px;
+`
+
+const UnlockUserDetails = styled('div')`
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 30px;
+  transition: transform 300ms ease-in-out;
+`
 
-  _onCreated = ({ id }, deposit, limitOfParticipants, coolingPeriod) => {
-    this.props.history.push(
+export default function Create() {
+  const [password, setPassword] = useState('')
+  const [locked, setLocked] = useState('pending')
+  const [membership, setMembership] = useState('')
+  const [membershipAddr, setMembershipAddr] = useState('')
+  let history = useHistory()
+
+  const _onCreated = ({ id }, deposit, limitOfParticipants, coolingPeriod) => {
+    history.push(
       `/deploy?id=${id}&deposit=${deposit}&limitOfParticipants=${limitOfParticipants}&coolingPeriod=${coolingPeriod}`
     )
   }
-}
 
-export default withRouter(Create)
+  const updateUnlockUser = () => {
+    // sets membership lock and lock name to state
+    let unlock = window.unlockProtocol.blockchainData()
+    let locks = Object.keys(unlock.locks).map(i => unlock.locks[i])
+    let bronze = locks.find(o => o.name.includes('Kickback Bronze'))
+    let gold = locks.find(o => o.name.includes('Kickback Gold'))
```

Comment:
Is this part used to determine which lock I became the member of?

<img width="313" alt="Screenshot 2019-12-17 at 22 36 48" src="https://user-images.githubusercontent.com/2630/71040128-cc164080-211d-11ea-8454-cd7a1b09a8bb.png">

Does `unlock.locks` returns all the locks only purchased by the specific Ethereum account? I tested with 4 different metamask accounts where account 1~3 bought gold and account 4 bought bronze though it kept showing gold no matter which ETH account I switch into.

When I put `console.log({locks})` under account 4 it had both bronze and gold hence looks like it picked gold automatically.

I wonder if `unlock.locks` just returns all the available locks, rather than the lock purchased by the purchaser (=event organiser). cc @julien51

Reply:
So, it is undocumented, but the way to get the locks for which the user has a key is to do this:

```
const data = window.unlockProtocol.blockchainData()
          if (data.keys) {
            // They should be indexed by lock.
            // Let's assume there is only one valid.
            const lockAddress = Object.keys(data.keys)[0]
          }
```

Reply:
> @makoto I wonder if unlock.locks just returns all the available locks, rather than the lock purchased by the purchaser (=event organiser).

You're right. 

> @julien51 So, it is undocumented, but the way to get the locks for which the user has a key is to do this: [...]

`data.keys` returned all available locks as well for me. Tested by purchasing on a new account.

I used `data.transaction` instead to get the latest purchase. Then displayed the membership by the lock purchased. I left the full algorithm commented in `updateUnlockUser()`.

Should display the correct membership owned now. 

<img width="655" alt="Screen Shot 2019-12-17 at 7 16 11 PM" src="https://user-images.githubusercontent.com/34966228/71045320-84ce8680-2102-11ea-9a00-fb3484248c30.png">


c0eaed3